### ISSUE TITLE
OpenRefrral: deploy / configure the test site for openreferral.localgovdrupal.org

### ISSUE THREAD

**Issue by finnlewis** [localgovdrupal/localgov_openreferral#6]

**Comment by ekes** [comment]
Documentation:-

```
# with localgov_openreferral module added; and localgov_directories on the feature/2.x/261-Open-Referral branch
> drush si localgov
> drush en localgov_demo
# now upgrade localgov_geo to feature/1.x/area-and-ux and run drush updb
> drush en localgov_directories_venue_or
# final step there's an open issue to decide the best way to get people to run this see localgovdrupal/localgov_directories#132
> drush eval "localgov_directories_venue_or_prepopulate_org();"
```

**Comment by ekes** [comment]
Outstanding bugs.

 - Taxonomy mapping hasn't installed. Install order?

**Comment by ekes** [comment]
It was installed, indexing was throwing the error fixed in 0d9c7932ac0f4c3cfdcf0e67eb9d40789ae5cefe
